src/Logger.js

```diff
@@ -23,7 +23,7 @@ class Logger {
   }
 
   write(message, persistent = false) {
-    if (!persistent) {
+    if (typeof message === 'string' && !persistent) {
```

Comment:
Maybe we should convert the message to a string instead? otherwise the lines will be wrong because we won't count them unless it is a string.

Reply:
Probably a good idea